### PR TITLE
Added surgicalKit treatment

### PR DESCRIPTION
**Pull request by BaerMitUmlaut**
This makes the surgical kit now actually do something. Simply clears the array of wounds that can break open again.

Although I'm not sure if a dedicated local function really is needed I have added it to follow the scheme of the other treatment functions.

### CODE REVIEW

**Comment by BaerMitUmlaut** [comment]
See #1284

**Comment by Glowbal** [comment]
I will expand this to support :
>dynamic treatment time as well.

And make it also do part of the work in case of cancel half way through. Unsure if I will be able to do this before the weekend.

Thanks @BaerMitUmlaut

**Comment by BaerMitUmlaut** [comment]
Dynamic treatment time has been implemented as well as a callbackProgress function instead. Left out the local function this time, I think it would only cause unnecessary network traffic.

File: addons/medical/functions/fnc_treatment.sqf [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 #include "script_component.hpp"
 
-private ["_caller", "_target", "_selectionName", "_className", "_config", "_medicRequired", "_items", "_locations", "_return", "_callbackProgress", "_treatmentTime", "_callerAnim", "_patientAnim", "_iconDisplayed", "_return", "_usersOfItems", "_consumeItems", "_condition", "_displayText", "_wpn"];
+private ["_caller", "_target", "_selectionName", "_className", "_config", "_medicRequired", "_items", "_locations", "_return", "_callbackProgress", "_treatmentTime", "_callerAnim", "_patientAnim", "_iconDisplayed", "_return", "_usersOfItems", "_consumeItems", "_condition", "_displayText", "_wpn", "_treatmentTime", "_treatmentTimeConfig"];
```
Comment (Glowbal) [inline addons/medical/functions/fnc_treatment.sqf]: `_treatmentTime` is twice in the private array

File: addons/medical/functions/fnc_treatmentAdvanced_surgicalKit_onProgress.sqf [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+_bandagedWounds = _target getVariable [QGVAR(bandagedWounds), []];
+
+//In case two people stitch up one patient and the last wound has already been closed we can stop already
+if (count _bandagedWounds == 0) exitWith {false};
+
+//Has enough time elapsed that we can close another wound?
+if ((_totalTime - _elapsedTime) <= (((count _bandagedWounds) - 1) * 5)) then {
+	_bandagedWounds deleteAt 0;
+	_target setVariable [QGVAR(bandagedWounds), _bandagedWounds, true];
```
Comment (Glowbal) [inline addons/medical/functions/fnc_treatmentAdvanced_surgicalKit_onProgress.sqf]: tabs. Please replace by 4 spaces

**Comment by Glowbal** [comment]
This has been merged. Thank you @BaerMitUmlaut

**Pull request merged**